### PR TITLE
feat : more filters in search doner feature

### PR DESCRIPTION
**Pull request by kewaldahal**

### CODE REVIEW

**Comment by kewaldahal** [comment]
HALA MADRID Y NADA MAS

File: src/mainApp.cpp [hunk 1]
```diff
@@ -201,7 +227,7 @@ class BloodDatabase {
                 }
             }
 
-            tempFile << d.donorId << "," << d.name << "," << d.address << "," << d.district << "," << d.bloodType << "," << d.number << endl;
+            tempFile << d.donorId << "  ,  " << d.name << "  ,  " << d.address << "  ,  " << d.district << "  ,  " << d.bloodType << "  ,  " << d.number << endl;
```
Comment (Bansnetsajak007) [inline src/mainApp.cpp]: refactor this more more fromatting is require

**Comment by kewaldahal** [comment]
ready to merge

**Comment by kewaldahal** [comment]
hala madrid'

**Comment by kewaldahal** [comment]
hala madrid

**Comment by kewaldahal** [comment]
merge it

**Comment by Bansnetsajak007** [comment]
Your code is shit, mate.

**Pull request merged**